src/main/java/com/fidesmo/fdsm/FidesmoCard.java

```diff
@@ -282,6 +282,20 @@ public boolean detect() throws CardException {
         return true;
     }
 
+    // Remove the leading 0x00 if the format is not TLV
```

Comment:
'_Leading_' or '_trailing_' TLV? Also, maybe we should check the `v` length?

Reply:
Good catch. Fixed!